src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -387,13 +388,23 @@ export default class CrossDeviceMobileRouter extends Component<
           token={this.state.token}
           fallback={<WrappedSpinner disableNavigation />}
         >
-          <HistoryRouter
-            {...this.props}
-            {...this.state}
-            crossDeviceClientError={this.setError}
-            sendClientSuccess={this.sendClientSuccess}
-            steps={steps}
-          />
+          <PoASupportedCountriesProvider
+            url={urls.onfido_api_url}
+            token={token}
```

Comment:
for orchestration, I guess this won't work as the token is not set as a property. @nulrich had a similar problem with the sdksettings. Please check back with him, how he solved it.

Reply:
@it-ony the endpoint with the PoA countries + document types needs the SDK token

Reply:
I spoke with @nulrich and made the change he suggested.